core/templates/pages/error-pages/error-page-root.component.spec.ts

```diff
@@ -16,13 +16,15 @@
  * @fileoverview Unit tests for error page root.
```

Comment:
Could we change the names of the old error pages to have "old" in them, to indicate that they are slated for deletion? E.g. old-error-page-root,component.spec.ts or similar.

Reply:
Done. The new one is just called `error-page.module.ts` but the old module and routing old have old prepended to them. The components have not been renamed because they are used for both old and new modules.